src/core/libraries/gnmdriver/gnmdriver.cpp

```diff
@@ -66,6 +66,9 @@ static inline u32* WriteTrailingNop(u32* cmdbuf) {
 
 s32 PS4_SYSV_ABI sceGnmAddEqEvent(SceKernelEqueue eq, u64 id, void* udata) {
     LOG_TRACE(Lib_GnmDriver, "called");
+    if (id != SceKernelEvent::Type::GfxEop) {
+        return ORBIS_OK;
```

Comment:
shouldn't return an error instead?

Reply:
Technically shouldn't, but we don't support them atm so just fake success for now